globals.d.ts

```diff
@@ -4,8 +4,11 @@ declare interface Web3 {
   sha3(str: string, options?: { encoding: 'hex' }): string;
 }
 
-declare interface Contract<T> {
+declare interface ContractBase {
   address: string;
+}
+
+declare interface Contract<T> extends ContractBase {
   deployed(): Promise<T>;
```

Comment:
`    'new'(options?: TransactionOptions): Promise<T>` should remove duplication in inheriting interfaces

Reply:
Yes, I can move the constructor to the base class `Contract<T>` but it would assume that all deriving contracts have zero-parameter constructors. At the moment this is true, but it will probably change. I'd rather not have zero-parameter constructor assumption in the base class.

Reply:
Obviously, I've missed that :)